suite-native/module-send-receive/src/screens/ReceiveAccountsScreen.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Box, Text, VStack } from '@suite-native/atoms';
+import { AccountsList } from '@suite-native/accounts';
+import {
+    AppTabsParamList,
+    AppTabsRoutes,
+    RootStackRoutes,
+    Screen,
+    ScreenHeader,
+    RootStackParamList,
+    TabToStackCompositeNavigationProp,
+} from '@suite-native/navigation';
+
+type ReceiveAccountsNavigationProps = TabToStackCompositeNavigationProp<
+    AppTabsParamList,
+    AppTabsRoutes.ReceiveScreen,
+    RootStackParamList
+>;
+
+export const ReceiveAccountsScreen = () => {
+    const navigation = useNavigation<ReceiveAccountsNavigationProps>();
+    return (
+        <Screen header={<ScreenHeader title="Receive" hasGoBackIcon={false} />}>
+            <Box flexDirection="column">
```

Comment:
Isn't this `Box` redudant here?

Reply:
Indeed it is. Fixed in [4eb63b8](https://github.com/trezor/trezor-suite/pull/7404/commits/4eb63b875dbf2fcbb850c0226e470ffaf0c7c9f4).